config/workflows/sa_gtd_workflow.json

```diff
@@ -73,10 +73,10 @@
                         }
                     ]
                 }, {
-                    "name": "comment_only",
+                    "name": "Comment_Only",
```

Comment:
I wonder if these names can do without the underscores and have clean looking meaningful names throughout. Should we try?